src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -119,7 +119,10 @@ private void GetDocuments(DocumentsOperationContext context, Transformer transfo
             }
             else if (HttpContext.Request.Query.ContainsKey("startsWith"))
             {
-                isLoadStartingWith = true;
+                // W.A. to distinguish between Lazily.LoadStartingWith of old client and LoadStartingWith of v4.0 new client
+                if (HttpContext.Request.Query.ContainsKey("ComingFromMultiGet") == false)
```

Comment:
W.A. ? 

Why are you doing something like that? Why is there different behavior between the clients?